WoWsShipBuilder.Common/wwwroot/css/app.css

```diff
@@ -234,7 +234,7 @@ div.mud-expand-panel {
 }
 
 .tab-panel-build-config-dialog {
-    max-height: calc(95vh - 150px);
+    max-height: calc(93vh - 78px);
```

Comment:
Use a different way to calculate this. The base value (view height) should be the max height of the dialog, then subtract the size that header and dialog actions take (at least 110px). "95vh - 110px" should work for pretty much all devices.